src/AspNet.Security.OAuth.Etsy/EtsyAuthenticationAccessType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+// for more information concerning the license and the contributors participating to this project.
+
+namespace AspNet.Security.OAuth.Etsy;
+
+public enum EtsyAuthenticationAccessType
+{
+    /// <summary>
+    /// Public client access type aka 'private usage access' in Etsy App Registration.
+    /// </summary>
+    Personal,
+
+    //// <summary>
+    //// Confidential client access type aka 'commercial usage access' in Etsy App Registration. // TODO: Uncomment if someone can verify that commercial usage access supports confidential clients.
+    //// </summary>
+    // Commercial
```

Comment:
I'd be reticent to actually ship this without this being validated as working. It would be better to just remove it entirely and add it later when needed and can be tested.